examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/EventHubService.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Consumer;
+using Azure.Messaging.EventHubs.Processor;
+using Azure.Storage.Blobs;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Microsoft.Extensions.Logging;
+
+namespace WebDemoWithEventHub
+{
+    public class EventHubService
+    {
+        private IConfigurationRefresher _refresher;
+
+        private ILogger<EventHubService> _logger;
+
+        public EventHubService(IConfiguration config, ILogger<EventHubService> logger)
+        {
+            _logger = logger;
+
+            EventHubConnection eventHubConnection = config.GetSection("WebDemo:EventHubConnection").Get<EventHubConnection>();
+            InitEventHubProcessor(eventHubConnection);
+            if (config is IConfigurationRoot configRoot)
+            {
+                foreach (var provider in configRoot.Providers)
+                {
+                    if (provider is Microsoft.Extensions.Configuration.AzureAppConfiguration.IConfigurationRefresher)
+                    {
+                        _refresher = (IConfigurationRefresher)provider;
```

Comment:
The refresher instance can now be obtained via DI

Reply:
Updated this to get the refresher instance through DI.

---
In reply to: [462049704](https://github.com/Azure/AppConfiguration/pull/329#discussion_r462049704) [](ancestors = 462049704)